code/backend/batch/utilities/helpers/EnvHelper.py

```diff
@@ -158,6 +158,10 @@ def __init__(self, **kwargs) -> None:
         self.LOAD_CONFIG_FROM_BLOB_STORAGE = self.get_env_var_bool(
             "LOAD_CONFIG_FROM_BLOB_STORAGE"
         )
+        # Integrated Vectorization
+        self.USE_INTEGRATED_VECTORIZATION = (
```

Comment:
I'm wondering if it's worth grouping this with the `AZURE_SEARCH` env vars? E.g. `AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION`. WDYT?

Reply:
yes it should be. Updated